src/qforte/utils/point_groups.py

```diff
@@ -0,0 +1,275 @@
+        0 : 'A  ',
+        1 : 'B  '
+    }
+
+    ci_to_int = {
+        'Ag ' : 0,
+        'Au ' : 1
+    }
+
+    ci_to_irrep = {
+        0 : 'Ag ',
+        1 : 'Au '
+    }
+
+    cs_to_int = {
+        'Ap ' : 0,
+        'App'  : 1
+    }
+
+    cs_to_irrep = {
+        0 : 'Ap ',
+        1 : 'App'
+    }
+
+    d2_to_int = {
+        'A  ' : 0,
+        'B1 ' : 1,
+        'B2 ' : 2,
+        'B3 ' : 3
+    }
+
+    d2_to_irrep = {
+        0 : 'A  ',
+        1 : 'B1 ',
+        2 : 'B2 ',
+        3 : 'B3 '
+    }
+
+    c2h_to_int = {
+        'Ag ' : 0,
+        'Bg ' : 1,
+        'Au ' : 2,
+        'Bu ' : 3
+    }
+
+    c2h_to_irrep = {
+        0 : 'Ag ',
+        1 : 'Bg ',
+        2 : 'Au ',
+        3 : 'Bu '
+    }
+
+    c2v_to_int = {
+        'A1 ' : 0,
+        'A2 ' : 1,
+        'B1 ' : 2,
+        'B2 ' : 3
+    }
+
+    c2v_to_irrep = {
+        0 : 'A1 ',
+        1 : 'A2 ',
+        2 : 'B1 ',
+        3 : 'B2 '
+    }
+
+    d2h_to_int = {
+        'Ag ' : 0,
+        'B1g' : 1,
+        'B2g' : 2,
+        'B3g' : 3,
+        'Au ' : 4,
+        'B1u' : 5,
+        'B2u' : 6,
+        'B3u' : 7
+    }
+
+    d2h_to_irrep = {
+        0 : 'Ag ',
+        1 : 'B1g',
+        2 : 'B2g',
+        3 : 'B3g',
+        4 : 'Au ',
+        5 : 'B1u',
+        6 : 'B2u',
+        7 : 'B3u'
+    }
+
+    if group == 'c1':
+        irrep_to_int = c1_to_int
+        int_to_irrep = c1_to_irrep
+    elif group == 'c2':
+        irrep_to_int = c2_to_int
+        int_to_irrep = c2_to_irrep
+    elif group == 'ci':
+        irrep_to_int = ci_to_int
+        int_to_irrep = ci_to_irrep
+    elif group == 'cs':
+        irrep_to_int = cs_to_int
+        int_to_irrep = cs_to_irrep
+    elif group == 'd2':
+        irrep_to_int = d2_to_int
+        int_to_irrep = d2_to_irrep
+    elif group == 'c2h':
+        irrep_to_int = c2h_to_int
+        int_to_irrep = c2h_to_irrep
+    elif group == 'c2v':
+        irrep_to_int = c2v_to_int
+        int_to_irrep = c2v_to_irrep
+    elif group == 'd2h':
+        irrep_to_int = d2h_to_int
+        int_to_irrep = d2h_to_irrep
+    else:
+        raise ValueError('The given point group is not supported. Choose one of:\nC1, C2, Ci, Cs, D2, C2h, C2v, D2h')
+
+    return irrep_to_int, int_to_irrep
+
+def char_table(group):
+
+    # Function that prints the character table of a chosen point group
+
+    group = group.lower()
+
+    ## Retrieve the dictionaries of irreps for the given point group
+    irrep_to_int, int_to_irrep = symmetry_irreps(group)
+
+    print('==========>', group, '<==========')
+    print('')
+    print('      ', end = '')
+
+    for irrep in irrep_to_int.items():
+        print(irrep[0], ' ', end = '')
+
+    print('\n')
+
+    for irrep1 in irrep_to_int.items():
+        print(irrep1[0], '  ', end = '')
+        for irrep2 in irrep_to_int.items():
+            print(int_to_irrep[irrep1[1] ^ irrep2[1]], ' ', end = '')
+        print()
+
+    return None
```

Comment:
```suggestion
```

Reply:
I know that by default the function will return `None`, I just thought that putting it there explicitly made the code more transparent. I will remove it.

Reply:
In this case, _not_ specifying a return value makes explicit that the point of the code isn't about a return value, but about something else. In this case, printing.